src/streaming/text/TextTracks.js

```diff
@@ -858,11 +858,6 @@ function TextTracks(config) {
         clearCaptionContainer.call(this);
     }
 
-    function deleteTextTrack(idx) {
```

Comment:
Why was this removed?

Reply:
It didn't necessarily need to be removed but I thought it was confusing as it didn't have the desired effect and was also unused. Trying to use this function to remove a track resulted in an error, which is why is disabled the track instead. This is because `addTextTrack()` is used on the video element to add text tracks. To reference [this page](https://pietropassarelli.gitbooks.io/textav/content/unconference-projects/removetexttrack-api.html) I came across:
> There are two ways to add new tracks: add a <track> element as a child of the video element, or use the video.addTextTrack API method.
These two methods are not the same. With the <track> element, you can remove the TextTrack by removing that <track> element. With video.addTextTrack, you cannot remove the created TextTrack.

Reply:
Thanks, makes sense to remove the function then